Assignment/SampleData.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Assignment;
+
+public class SampleData : ISampleData
+{
+    // 1.
+    /*
+        Property to read lines from the CSV file "People.csv". The first line (header) is skipped.
+        This uses File.ReadLines to avoid loading the entire file into memory, which is useful for large files.
+    */
+    public IEnumerable<string> CsvRows { get; set; } = File.ReadLines("People.csv").Skip(1);
+
+    // 2.
+    public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows()
```

Comment:
Doesn't affect the functionality at all, but you could potentially change this to a lambda expression,  and avoid using the "return" statement. 
```suggestion
    public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows() => CsvRows.Select(row => row.Split(",")[6].Trim())
    .Distinct()
    .Orderby(state => state);
```

Reply:
Ok I'm about to work on this

Reply:
Ok the code has been updated